modules/olm-operatorgroups-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="olm-operatorgroups-limitations"]
+= Limitations for multi-tenant Operator management
+
+OpenShift Container Platform provides limited support for installing different variations of an Operator on a cluster at the same time. Operators are control plane extensions. Since all tenants, or namespaces, share the same control plane of a cluster, tenants in a multi-tenant environment also have to share Operators
+
+Operator Lifecycle Manager (OLM) installs Operators multiple times in different namespaces and one of the constraints being that the Operator’s API versions need to be the same
+
+Different major versions of an Operator often have incompatible custom resource definitions (CRDs). This makes it difficult to quickly verify OLMs
+
+.Additional resources
```

Comment:
I think xrefs still need to be in assemblies. So you may want to move this Additional Resources section to an assembly.

Reply:
See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies

Reply:
Also missing a period This makes it difficult to quickly verify OLMs.<<